nss-day-cohort-44/nutshell-labyrinth-leprechauns
# Sl chat start

# Description
Chat is operational. Saves and posts messages.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# Testing Instructions
Login as a user. Type message and post. it should display on the dom and save to the api.
# Checklist:
